src/collection-facets.ts

```diff
@@ -490,6 +580,28 @@ export class CollectionFacets extends LitElement {
         opacity: 0.5;
       }
 
+      /* add the following styles to ensure proper modal visibility */
+      body.modal-manager-open {
```

Comment:
? top level body? this may be too high up. let's move this to app root?

Reply:
Yeah!